src/edu/csus/ecs/pc2/ui/TestResultsPane.java

```diff
@@ -1894,6 +1894,44 @@ private void addAnyUnexecutedTestCasesToResultsTable(JTable aResultsTable) {
             }
         }
     }
+
+    /**
+     * Clears the given JTable with empty test cases
+     * 
+     * @param aResultsTable - a JTable expected to already contain one row for each test case which has been executed
+     */
+    public void resetResultsTable() {
+        if(resultsTable != null) {
+            //get the table model which defines the current table contents
+            TestCaseResultsTableModel tableModel = (TestCaseResultsTableModel) resultsTable.getModel();
+            
+            //get how many total test cases were configured into the problem
+            int totalTestCaseCount = currentProblem.getNumberTestCases();
```

Comment:
this seems unused?

Reply:
Yes.  Lazy duplication of code from addAnyUnexecutedTestCases()...  It has been removed and pushed.